Doc/library/http.server.rst

```diff
@@ -439,6 +439,15 @@ the following command uses a specific directory::
 .. versionadded:: 3.7
     ``--directory`` specify alternate directory
 
+By default, server supports HTTP up to version HTTP/1.0. The option ``-p/--protocol``
```

Comment:
```suggestion
By default, the server supports HTTP up to version HTTP/1.0. The option ``-p/--protocol``
```

Reply:
I was following the wording of the description of the other CLI options, but now I see that you have fixed them in this [PR](https://github.com/python/cpython/commit/c8a536624e8f5d6612e3c275c5b19592583a8cf8). So thanks, updated!